eixampledb-core/src/main/java/com/eixampledb/core/EixampleDbRunnerService.java

```diff
@@ -21,37 +19,38 @@
 
     @Override
     public GetResponse get(GetRequest getRequest) {
```

Comment:
That is a refactor that is not related with the pull request. But there was too much code repeated.